tests/testthat/render-figs.R

```diff
@@ -0,0 +1,32 @@
+library(ggExtra)
+library(ggplot2)
+library(vdiffr)
+library(fontquiver)
+library(svglite)
+
+# Load functions that will be used to create the figures
+source("tests/testthat/helper-funs.R")
+
+dir <- "tests/figs/ggMarginal/ggplot2-2.2.1"
```

Comment:
I was thinking of adding `ggplot2Version` an argument to `writeSVG`, which will create `dir` if it doesn't exist for a particular version of ggplot2